co-simulation tool/README.md

```diff
@@ -0,0 +1,418 @@
+
+
+# Carla-Sumo-Mosaic Co-Simulation Tool
+
+This co-simulation tool is used to run [CARLA simulator](http://carla.org/) and [SUMO simulator](https://www.eclipse.org/sumo/) with [Eclipse MOSAIC framework](https://github.com/eclipse/mosaic). The co-simulation tool can simulate both co-simulation scenarios and original Eclipse MOSAIC scenarios on both Windows systems and Linux systems.  
+
+##  **Prerequisites**
+
+For a successful running of the co-simulation tool, the following software must be installed:
+
+-  CARLA simulator
+-  JAVA 8 or 11 - Recommend using the [Adopt OpenJDK](https://adoptopenjdk.net/?variant=openjdk8&jvmVariant=hotspot)
+-  Python 3.x - The default python version is 3.7. The python version should be the same as the one used by CARLA Python API. If the users use different version of Python other than 3.7, the users need to copy the CARLA library .egg file to the folder bridge/ PythonAPI /carla/dist. In addition, the package `lxml==4.5.0` needs to be installed using `pip3 install lxml==4.5.0`. 
+-  SUMO 1.8.0 - The environment variable **`SUMO_HOME`** **must be configured to define the installation directory of SUMO**.
+
+Moreover, a high performance computer is needed to run the co-simulation tool. The configuration of a recommended system is in the following.
+
+- Intel i7 gen 9th - 11th / Intel i9 gen 9th - 11th / AMD ryzen 7 / AMD ryzen 9
+- +16 GB RAM memory
+- NVIDIA RTX 2070 / NVIDIA RTX 2080 / NVIDIA RTX 3070, NVIDIA RTX 3080
+
+## Installation of Co-simulation Tool
+
+### **a.**   **Executable Bundle File**
+
+Download the CARLA-SUMO-MOSAIC-1.0.zip bundle and extract it to an arbitrary path. 
+
+### **b.**   **Installation from Source Code**
+
+The co-simulation tool has been built successfully on different Operating Systems (OS). The building results are shown in the following table.
+
+| Test NO. | OS          | OS Version   | Java JDK                | Maven | SUMO  | Building Results |
```

Comment:
Please add the testing CARLA version in the form.

Reply:
When building the bundle file of co-simulation tool from the source code, CARLA is not needed.

Reply:
If the build package is not tested in the environment you mentioned in this form, please do not include this form here. Building successfully does not mean running successfully. It is confusing.

Reply:
The form has been removed from README.md.